Documentation/gettingstarted/k8s-install-etcd-operator.rst

```diff
@@ -116,25 +116,50 @@ reverse lookup on a pod IP must map back to pod name. If you are using CoreDNS,
 check the CoreDNS ConfigMap and validate that ``in-addr.arpa`` and ``ip6.arpa``
 are listed as wildcards for the kubernetes block like this:
 
-::
-
-    kubectl -n kube-system edit cm coredns
-    [...]
-    apiVersion: v1
-    data:
-      Corefile: |
-        .:53 {
-            errors
-            health
-            kubernetes cluster.local in-addr.arpa ip6.arpa {
-              pods insecure
-              upstream
-              fallthrough in-addr.arpa ip6.arpa
-            }
-            prometheus :9153
-            proxy . /etc/resolv.conf
-            cache 30
-        }
+    .. tabs::
+        .. group-tab:: Kubernetes 1.16+
+
+            ::
+
+                kubectl -n kube-system edit cm coredns
+                [...]
+                apiVersion: v1
+                data:
+                  Corefile: |
+                    .:53 {
+                        errors
+                        health
+                        kubernetes cluster.local in-addr.arpa ip6.arpa {
+                          pods insecure
+                          upstream
+                          fallthrough in-addr.arpa ip6.arpa
+                        }
+                        prometheus :9153
+                        forward . /etc/resolv.conf
+                        cache 30
+                    }
+
+        .. group-tab:: Kubernetes < 1.16
+
+            ::
+
+                kubectl -n kube-system edit cm coredns
+                [...]
+                apiVersion: v1
+                data:
+                  Corefile: |
+                    .:53 {
+                        errors
+                        health
+                        kubernetes cluster.local in-addr.arpa ip6.arpa {
+                          pods insecure
+                          upstream
+                          fallthrough in-addr.arpa ip6.arpa
+                        }
+                        prometheus :9153
+                        forward . /etc/resolv.conf
```

Comment:
Isn't the point of the tabs to leave this one at `proxy`?

Reply:
opps :) copy and paste problem. Thanks :100: